docs/notebooks/platform_integration.ipynb

```diff
@@ -26,10 +26,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Glossary\n",
-    "A list of useful keywords:  \n",
+    "## Useful keywords\n",
     "\n",
-    "- **Credo AI Platform**: Also referred to as simply \"platform\". The central AI governance/reporting Platform, found at [https://app.credo.ai/](https://app.credo.ai/)\n",
+    "- **Credo AI Platform**: Also referred to as simply \"Platform\". The central AI governance/reporting Platform, found at [https://app.credo.ai/](https://app.credo.ai/)\n",
     "\n",
     "- **credoconfig**: configuration file to be copied in the user's home folder\n",
     "\n",
```

Comment:
Line 39 -> "(where do we get this?)

Either answer that question or delete (ideally the first)